tests/storages_tests/rdb_tests/test_with_server.py

```diff
@@ -115,3 +134,26 @@ def test_multiprocess(storage_url: str) -> None:
     assert len(trials) == n_workers * 20
 
     _check_trials(trials)
+
+
+@pytest.mark.parametrize("n_jobs", [1, 4])
+def test_multiprocess_with_progbar(storage_url: str, n_jobs: int) -> None:
+    n_workers = 8
+    study_name = _STUDY_NAME
+    optuna.create_study(storage=storage_url, study_name=study_name)
+    with Pool(n_workers) as pool:
+        results = pool.map(
```

Comment:
`results` is a little bit too general. How about this?
```suggestion
        progress_bars_as_str = pool.map(
```

Reply:
Sorry, this variable was unclear (retuned value is not str). I'll revise the comment above.

Reply:
Ok, fixed.